engine/src/db/mod.rs

```diff
@@ -5,16 +5,17 @@ use sqlx::{migrate::MigrateDatabase, Pool, Sqlite, SqlitePool};
 pub mod handles;
 pub mod models;
 
-use crate::utils::db_path;
+use crate::utils::DB_PATH;
 
 pub async fn db_pool() -> Result<Pool<Sqlite>, sqlx::Error> {
-    let db_path = db_path().unwrap();
+    let db_path = DB_PATH.as_ref().unwrap();
+    let db_path = db_path.to_string_lossy();
 
-    if !Sqlite::database_exists(db_path).await.unwrap_or(false) {
-        Sqlite::create_database(db_path).await.unwrap();
+    if !Sqlite::database_exists(&db_path).await.unwrap_or(false) {
+        Sqlite::create_database(&db_path).await.unwrap();
```

Comment:
If we already here we can also replace unwrap() with ? :-)

Reply:
I wanted to stay as close to the previous behaviour outside of `db_path()` as possible, but I can refactor this as well